native/packages/react-native-bpk-component-button/src/BpkButton.js

```diff
@@ -161,7 +225,13 @@
    selected: false,
    accessibilityLabel: null,
    style: null,
+   theme: null,
  };
 
+ const BpkButtonThemed = withTheme(BpkButton);
```

Comment:
Originally I wanted to just export only `BpkButton` and have it take theming if it's inside a `BpkThemeProvider`. This doesn't work, because `withTheme` throws an error if it's not inside `BpkThemeProvider`, so it won't work for non-themed buttons.

Assuming we can't find a way around this, I think we should standardise on how to export themed variants of components.